packages/pwa-kit-dev/src/ssr/server/public-path.js

```diff
@@ -17,4 +17,6 @@
 import {bundleBasePath} from '@salesforce/pwa-kit-runtime/utils/ssr-namespace-paths'
 
 /* global __webpack_public_path__: writable */
+// This global variable is used by webpack for HMR (Hot Module Replacement)
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
In general we don't want to see changes in the `pwa-kit-dev` package for the Shopper Agent functionality unless I'm missing something